src/core/Akka/Actor/ActorSystem.cs

```diff
@@ -74,6 +74,24 @@ public abstract class ActorSystem : IActorRefFactory, IDisposable
         public abstract ILoggingAdapter Log { get; }
 
         /// <summary>
+        /// Start-up time in milliseconds since the epoch.
+        /// </summary>
+        public long StartTime => _startTime;
+
+        private readonly long _startTime = MillisecondsSinceEpoch();
+
+        /// <summary>
+        /// Up-time of this actor system in seconds.
+        /// </summary>
+        public long Uptime => (MillisecondsSinceEpoch() - StartTime) / 1000;
+
+        /// <summary>
+        /// Returns the time in milliseconds since the epoch
+        /// </summary>
+        private static long MillisecondsSinceEpoch()
```

Comment:
You should use the methods we have in the `MonotonicClock` under utils and see if that suits your needs here. If it doesn't, then yes you should move this into a utility class in that namespace.